mesos/framework/cluster.go

```diff
@@ -19,11 +19,12 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
-	"github.com/elodina/go-mesos-utils"
 	"sort"
 	"strconv"
 	"strings"
 	"sync"
+
+	utils "github.com/elodina/go-mesos-utils"
```

Comment:
@joestein you accidentally pinged wrong person (miry I mean) :) Pinged @g0dlike.
The go-mesos-utils branch should also get test coverage before it gets merged, didn't have a chance to cover it with tests yet. Also I think we will add a few more things so it makes sense to wait a little bit before merging that branch.